app/services/cocina_object_store.rb

```diff
@@ -0,0 +1,13 @@
+class CocinaObjectStore
+  def self.find(druid)
+    path = "#{purl_druid_path(druid)}/cocina.json"
+    data_hash = JSON.parse(File.read(path))
+    Cocina::Models.build(data_hash)
```

Comment:
So let's suppose we're rolling out a breaking cocina change and re-publishing all items. What happens if we publish an object which requires loading objects that have not yet been republished? That is, are we requiring a republishing order?

Reply:
Only in the case of Virtual objects, where the constituents need to be published before the object that is composed of the constituents.